elephant/utils.py

```diff
@@ -372,3 +394,54 @@ def get_opencl_capability():
         return True
     except ImportError:
         return False
+
+
+def trials_to_list_of_spiketrainlist(method):
+    """
+    Decorator to convert Trials object to a list of spiketrainlists before
+    calling the wrapped method.
+
+    Parameters
+    ----------
+    method: callable, The method to be decorated.
+
+    Returns
+    -------
+    callable: The decorated method.
+
+    Raises
+    ------
+    ValueError: If the number of trials in the Trials object is 0.
+
+    Example:
```

Comment:
The example section also has a specific header

Reply:
Fixed.